### DISCUSSION TITLE
[ Ollama Remote ] where do I specify a model?

### DISCUSSION THREAD

**Discussion by MrXandbadas** [Mobile-Artificial-Intelligence/maid#592] [Q&A]
Hello there, in the model setting I can put in a key, a url but can't select a model or don't know how to do so
![Screenshot_20240720-150505.png](https://github.com/user-attachments/assets/0f9db74a-fb47-46cb-8d16-05f6ff9608af)

![Screenshot_20240720-150535.png](https://github.com/user-attachments/assets/33688847-cf4a-4703-9be2-52fd08c1f9e3)

**Comment by danemadsen** [comment]
Currently you have to download a model on the PC running ollama first and then it will show up in maid.

**Reply by ueng03295** [reply]
but I meet this even if I run a model in ollama locally
